cmd/root.go

```diff
@@ -78,6 +114,29 @@ func Execute() {
 	}
 }
 
+type advancedConfig struct {
+	Env          []corev1.EnvVar
+	Volumes      []corev1.Volume      `json:"volumes"`
+	VolumeMounts []corev1.VolumeMount `json:"volumeMounts"`
+}
+
+func processAdvancedConfig(cmd *cobra.Command, args []string) error {
```

Comment:
This is for verification that the config is correct, right?

Reply:
Before it validated is the file was valid JSON, nothing more. Now it does field-level validation via https://github.com/dominodatalab/forge/pull/42/commits/9f51d6575540f0521cb5503b9ed8c3b768e21123